src/lib/variants.test.ts

```diff
@@ -164,6 +169,88 @@ describe('variant filters', () => {
         expect(got2).toBe(false);
     });
 
+    it('should filter by articles viewed settings', () => {
+        // Test 1 - below min articles viewed
```

Comment:
Thoughts on splitting this into multiple `it` blocks, one for each of the comments/assertions? E.g. `it('filters correctly when below the min articles viewed', ...` etc.

Reply:
Yeah I mentioned this whilst catching you with you and Nic yesterday. I've followed the same pattern already in place, which is one test (with multiple assertions) per filter. My preference would be to consider making each test a separate file, with its own set of tests. But until we get there I'd rather keep the same pattern we have on that file for all the other filters.

Reply:
BTW I agree with you, I like one assertion per `it` block, but if we do that we should to the same to all other tests and that's outside of the scope of this PR.

Reply:
I've carded this up to pick up separately - https://trello.com/c/jxolLo9O/80-refactor-variant-tests-for-readability